client/src/Components/Common/homePage/Home.js

```diff
@@ -1,25 +1,85 @@
-import React, { Component } from "react";
-import { withStyles } from "@material-ui/core/styles";
-import { Grid, Box } from "@material-ui/core";
+import React, { useEffect, useState } from "react";
+import { Grid } from "@material-ui/core";
 import ArtCard from "../artCard/ArtCard";
+import axios from "axios";
+import RecentArtSlider from "../RecentArtSlider/RecentArtSlider";
+import Header from "../Header/Header";
+import Navbar from "../Navbar/Navbar";
+import Categories from "../categoriesSlider/CategoriesSlider";
+import Typography from "@material-ui/core/Typography";
+import useStyles from "./style";
+import ArrowBackIcon from "@material-ui/icons/ArrowBack";
+import green from "@material-ui/core/colors/green";
+import customizePhoto from "./customize.png";
 
 function Home() {
-
+  const classes = useStyles();
+  const [artwork, setArtwork] = useState(null);
+  useEffect(() => {
+    if (artwork) {
+      return;
+    }
+    axios
+      .get("/api/get-art")
+      .then((result) => setArtwork(result.data))
+      .catch((err) => console.log(err));
```

Comment:
Might consider a logging abstraction. It could even just wrap console.log, for now, but the idea would be to helps to avoid "debugging artifacts" that might sneak into commits, and makes it easier to add more dynamic error telemetry in future.